.clang-tidy

```diff
@@ -8,7 +8,6 @@ boost-*,
 bugprone-*,
 -bugprone-easily-swappable-parameters,
 -bugprone-exception-escape,
--bugprone-macro-parentheses,
```

Comment:
cool! i guess this was resolved in the metadata refactoring by tony

Reply:
can you also check the other clang tidy issues listed in the aliases in https://clang.llvm.org/extra/clang-tidy/checks/list.html ?